### PR TITLE
#162163223 Set up react router version 4

### PR DESCRIPTION
**Pull request by kafuuma**
#### What does this PR do?
This PR delivers a chore to implement react router 4 for the application

#### Description of the tasks to be completed?
- Components can be accessed by the corresponding URLs
- Components that are linked to routes are placed in the views directory
- Trying to accessing a URL that doesn't exist redirects to a page not found 

#### URLS
```
-  "/ "           - login page
- "/signup"  - Signup page

```

Screenshots 

1. Not Found Page

pagenotfound

2. LSignup URL
signup

#### What are the relevant pivotal tracker stories

[#162163223](https://www.pivotaltracker.com/story/show/162163223)

### CODE REVIEW

**Review by bibangamba — APPROVED** [review]

**Review by joshtrigger — APPROVED** [review]
nice work just remember to please add the ```.vscode folder``` to `gitignore` file

File: src/Router.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import NotFound from './views/NotFound'
+import Signup from './views/auth/signup/Signup'
+import Login from './views/auth/Login/Login'
+
+const Routes = () =>(
+    <Router>
+        <Switch>
+            <Route path='/' component={Login} exact />
+            <Route path='/signup' component={Signup} exact />
+            {/* <Route path='/login' component={Login} exact /> */}
```
Comment (JaymesKat) [inline src/Router.js]: @kafuuma please remove this comment.

**Review by JaymesKat — CHANGES_REQUESTED** [review]
@kafuuma Good job, some changes to consider.

add the `styles` for each `view/component` next to it. In this case, place `notFound.scss` next to `NotFound.js`. It is a convention that we will be using throughout the project for better reusability.

Also, rename all files that include `JSX` with a .jsx extension as per the conventions [here](https://github.com/airbnb/javascript/tree/master/react#naming)

**Review by Kasulejoseph — APPROVED** [review]

File: src/Router.js (hunk 1 above)
Comment (kafuuma) [reply src/Router.js]: I have removed it

**Review by JaymesKat — APPROVED** [review]
LGTM

**Pull request merged**